Mage.Sets/src/mage/cards/s/SharedFate.java

```diff
@@ -63,6 +63,7 @@ public SharedFate(UUID ownerId, CardSetInfo setInfo) {
 
         // Each player may look at and play cards he or she exiled with Shared Fate.
         this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new SharedFatePlayEffect()));
+        this.addAbility(new SimpleStaticAbility(Zone.ALL, new SharedFateLookEffect()));
```

Comment:
Unless the text of a card explicitly states or implies otherwise abilities on cards only function on the battlefield.